e2e-tests/functions

```diff
@@ -80,7 +80,7 @@ deploy_operator() {
 	yq r -d'2' ${src_dir}/deploy/operator.yaml 'data[values.yaml]' \
 		| $sed -e "s#namespace: .*#namespace: ${namespace}#g" \
 		| $sed -e "s#pgo_operator_namespace: .*#pgo_operator_namespace: ${namespace}#g" \
-		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: ${GIT_BRANCH}#g" \
+		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: $(echo ${IMAGE_OPERATOR%-operator-postgres} | cut -d: -f2)#g" \
```

Comment:
```suggestion
		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: $(echo ${IMAGE_OPERATOR%-postgres-operator} | cut -d: -f2)#g" \
```
postgres-operator ?

Reply:
Fixed
worked well in https://cloud.cd.percona.com/view/PGO/job/pgo-operator-gke-version/16/